src/client/components/PageLayout/Footer/hooks/useFooterLogic.ts

```diff
@@ -0,0 +1,15 @@
+import { useMemo } from 'react'
+
+import { useIsPrintRoute } from 'client/hooks/useIsRoute'
+import { useCycleRouteParams } from 'client/hooks/useRouteParams'
+
+export const useFooterLogic = () => {
```

Comment:
can we call it useFooterVisible?

Reply:
Since it now just returns a boolean, I will keep it simple:
```
export const useIsFooterVisible = (): boolean => {

{...}

return isFooterVisible
```